### DISCUSSION TITLE
Can´t open

### DISCUSSION THREAD

**Discussion by DinoVikingo** [anidl/multi-downloader-nx#144] [Q&A] [answered]
Hi!
How can i use this?
When i click on the .exe opens a CMD window and immediatly closes.
![image](https://user-images.githubusercontent.com/89960654/140564327-3099a996-0a82-491f-97d9-10e5ca72d9ac.png)

**Comment by izu-co** [comment, marked as answer]
Hey there.

Please post an issue for these requests.
You have to open a cmd navigate into the folder and then write aniDL.exe.
